### ISSUE TITLE
[Request] Add Encrypted exports + imports

### ISSUE THREAD

**Issue by JSKitty** [PIVX-Labs/MyPIVXWallet#24]
To enhance the security of MPW, we should start *discouraging the use of naked WIF private keys* in day-to-day use, by allowing full use of our existing AES-GCM encrypted MPW keys.

In theory this means an update to the password manager which either:
- **Easier:** A pattern to recognise AES-GCM keys and switch to the 'Enter Password' mode.
- **Harder:** Improve our AES keys by prefixing them with a very specific byte serial, so we can guarantee an input is 'WIF' or 'AES-WIF', this also requires **carefully** updating existing keys, with full backwards-compat, and then again switching to 'Enter Password' mode upon key import.

Using a unique byte serial prefix gives MPW keys a distinct verifiability (a deterministic key type, so we can verify: prefix, salt, IV, and encrypted bytes), and thus is a much preferred method to 'guessing' if a key is encrypted.

After this, we should add a dedicated / refined UI to 'Export' encrypted keys either to the Clipboard or directly to a custom MPW file, which could be used as an import file in addition to the copy/paste import method.

## TL;DR recap:
A developer here, should:
- Define and write a method of detecting if a key if plain WIF or AES-WIF.
- Import said key to storage, switch to 'Unlock Wallet' mode.
- Create an Export UI which supports two options: Clipboard or File.

If anyone picks this task up, please ping me here, I'll gladly assist, and a Labs bounty is attached as per the GitHub labels.

**Comment by JSKitty** [comment]
Currently assigned to @Shangaman (GitHub is being strange and not letting me assign unless tagged/commented!)
